packages/e2e-test-utils/src/press-key-with-modifier.js

```diff
@@ -126,6 +126,7 @@ async function emulateClipboard( type ) {
 		document.activeElement.dispatchEvent(
 			new ClipboardEvent( _type, {
 				bubbles: true,
+				cancelable: true,
```

Comment:
When the event option is not specified [cancelable](https://developer.mozilla.org/en-US/docs/Web/API/Event/cancelable) defaults to false.